File: VersaoFinal/Latex/codigos/micro01.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
 
namespace CelParaFar
{
    class Programa
    {
        static void Main(string[] args)
        {
            int cel, far;
            Console.WriteLine("Tabela de conversão: Celsius -> Fahrenheit");
            Console.WriteLine("Digite a temperatura em Celsius: ");
            cel = int.Parse(Console.ReadLine());
            far = (cel * 9) / 5 + 32;
            Console.WriteLine("A nova temperatura é: " + far +" F");
      
        }
    }
}

File: CC/trabalho8/final/final/exemplosLua/micro01.lua
-- Listagem 13: Converte graus Celsius para Fahrenheit

-- [[ Função: Ler uma temperatura em graus Celsius e apresentá-la convertida em graus Fahrenheit. A fórmula de conversão é : F=(9*C+160) / 5, sendo F a temperatura em Fahrenheit e C a temperatura em Celsius. --]]

print("Tabela de Conversão: Celsius -> Fahrenheit")
print("Digite a Temperatura em Celsius: ")
cel = io.read("*number")
far = (9*cel+160)/5
print("A nova temperatura é:", far)